app/views/simplified-account/settings/email-notifications/collect-email-page.njk

```diff
@@ -0,0 +1,62 @@
+{% extends "../settings-layout.njk" %}
```

Comment:
nunjucks templates shouldn't have a `-page` suffix. Something like `edit-email-collection-mode.njk` would be more appropriate.

Reply:
As per this [comment](https://github.com/alphagov/pay-selfservice/pull/4306#discussion_r1820661981) I'll want to change this on the next cypress PR instead, as I already have the cypress tests working and in a different branch.